docker/run_backend.sh

```diff
@@ -17,12 +17,12 @@ export ROS_HOSTNAME=$ROS_IP
 
 # Function to stop the container on exit sig_(exit int term)
 function cleanup {
-    docker-compose -f $HOME/catkin_ws/src/rasberry_perception/docker/docker-compose.yml rm -sf $SERVICE_NAME
+    docker-compose -f docker-compose.yml rm -sf $SERVICE_NAME
```

Comment:
I thought we wanted to get rid of the path completely. By placing the run_backend.sh in the same directory with docker-compose.yaml, we can just remove the `-f` flag completely:
```
docker-compose rm -sf $SERVICE_NAME
```
and 
```
docker-compose run $SERVICE_NAME
```

Reply:
ROS, in order to be able to find this file, we should modify the very end of the CMakeLists.txt
from:
```
file(GLOB python_scripts_DIR "scripts/*.py")
file(GLOB bash_scripts_DIR "scripts/*.sh")
catkin_install_python(PROGRAMS ${python_scripts_DIR} ${bash_scripts_DIR} DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
```
to
```
file(GLOB python_scripts_DIR "scripts/*.py")
file(GLOB bash_scripts_DIR "scripts/*.sh")
file(GLOB docker_scripts_DIR "docker/*.sh")
catkin_install_python(PROGRAMS ${python_scripts_DIR} ${bash_scripts_DIR} ${docker_scripts_DIR} DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
```

Reply:
hmm, then why does it work without doing this?